.github/workflows/test.yml

```diff
@@ -3,12 +3,15 @@ name: Test Action
 on:
   workflow_dispatch:
```

Comment:
This should be updated to pull_requests and on push to main. So that this workflow is run in the PR